lib/opensusebasetest.pm

```diff
@@ -440,7 +440,7 @@ sub wait_boot {
         handle_emergency if (match_has_tag('emergency-shell') or match_has_tag('emergency-mode'));
 
         if (!$nologin) {
-            if (get_var('DM_NEEDS_USERNAME')) {
+            if (get_var('DM_NEEDS_USERNAME') || check_var('VERSION', '11-SP4')) {
```

Comment:
Instead updating the if condition, could we set DM_NEEDS_USERNAME for SLE 11-SP4 to get the same behavior?

Reply:
yes, it shoul be get the same behavior, but DM_NEEDS_USERNAME should be aslo used in other test cases, I don't think it is safe to use it directly.

Reply:
Add it back, since need modified more pm with set 'DM_NEEDS_USERNAME'.